dms/static/src/xml/views.xml

```diff
@@ -57,24 +57,21 @@
                     t-attf-src="https://www.youtube.com/embed/#{widget.activeAttachment.youtube}"
                 />
                 <video
-                    t-if="widget.activeAttachment.fileType == 'video'"
+                    t-if="widget.activeAttachment.type == 'video'"
                     class="o_viewer_video"
                     controls="controls"
                 >
                     <source
-                        t-attf-src="/web/image/#{model}/#{widget.activeAttachment.id}/#{widget.fieldName}"
+                        t-attf-src="/web/image/#{widget.modelName}/#{widget.activeAttachment.id}/#{widget.fieldName}"
                         t-att-data-type="widget.activeAttachment.mimetype"
                     />
                 </video>
             </div>
         </t>
     </t>
     <t t-name="FieldBinaryPreview" t-extend="DocumentViewer">
-        <t t-jquery=".o_document_viewer_content_call" t-operation="replace">
-            <t
-                class="o_document_viewer_content_call"
-                t-call="FieldBinaryPreview.Content"
-            />
+        <t t-jquery="[t-call='DocumentViewer.Content']" t-operation="replace">
```

Comment:
A replace operation is not usually advisable. Isn't there any other option?

Reply:
I know that it's a replace, but the new element inherits from DocumentViewer.Content, It was the only way I found easy enough in order to change the logic :thinking:

Reply:
OK if you don't see a better approach, but always remember to add a comment explaining the whys for better later comprehension.